auth/auth-server/src/main.rs

```diff
@@ -39,6 +44,12 @@ use server::Server;
 
 /// The default internal server error message
 const DEFAULT_INTERNAL_SERVER_ERROR_MESSAGE: &str = "Internal Server Error";
+/// The dummy private key used to instantiate the arbitrum client
+///
+/// We don't need any client functionality using a real private key, so instead
+/// we use the key deployed by Arbitrum on local devnets
+const DUMMY_PRIVATE_KEY: &str =
```

Comment:
i'll add this to my todo to remove the need for a private key. then we can take the dummy values out of our workers